msteams-platform/graph-api/activity-feed/feed-notifications.md

```diff
@@ -0,0 +1,171 @@
+
+You'll need to have the following values from the Azure AD registration process to retrieve an access token from the identity platform:
+
+- The **Application ID** assigned by the app registration portal. If your app supports single sign-on (SSO) you should use the same Application ID for your app and SSO.
+- The  **Client secret/password** or a public/private key pair (**Certificate**). This is not required for native apps.
+- A **Redirect URI** (or reply URL) for your app to receive responses from Azure AD.
+
+ *See* [Get access on behalf of a user](/graph/auth-v2-user?view=graph-rest-1.0#3-get-a-token) and [Get access without a user](/graph/auth-v2-service).
+
+## Update your Teams app manifest
+
+> [!div class="checklist"]
+> [!div class="checklist"]
+> - Add a [webApplicationInfo](../../resources/schema/manifest-schema.md#webapplicationinfo) key to your app manifest with the following values:  
+&emsp;&emsp; &#8226; **id**  — your Azure AD app id.  
+&emsp;&emsp; &#8226;  **resource** — the resource url for the app.
+>
+> [!div class="checklist"]
+> [!div class="checklist"]
+>
+> - Add an **activities** key to your app manifest with an `activityType array`  value. Each activity type object will have the following properties:  
+&emsp;&emsp; &#8226; **type**  
+&emsp;&emsp; &#8226;  **description**  
+&emsp;&emsp; &#8226;  **templateText**
+
+```json
+"webApplicationInfo": {
+    "id": "73ee2834-38aa-4077-b4c9-e8e1c36f7e40",
+    "resource": "https://AkjStoreApp"
+  },
+  "activities": {
+    "activityTypes": [
+      {
+        "type": "taskCreated",
+        "description": "Task Created Activity",
+        "templateText": "{actor} created task {taskId} for you"
+      },
+      {
+        "type": "teamMention",
+        "description": "Team Mention Activity",
+        "templateText": "{actor} mentioned random team"
+      },
+      {
+        "type": "channelMention",
+        "description": "Channel Mention Activity",
+        "templateText": "{actor} mentioned random channel"
+      },
+      {
+        "type": "userMention",
+        "description": "Personal Mention Activity",
+        "templateText": "{actor} mentioned you dude"
+      },
+      {
+        "type": "calendarForward",
+        "description": "Forwarding a calendar event",
+        "templateText": "{actor} sent you an invite on behalf of {eventOwner}"
+      },
+      {
+        "type": "creatorTaskCreated",
+        "description": "Created Task Created",
+        "templateText": "The Creator created task {taskId} for you"
+      }
+    ]
+  },
+```
+
+## Activity feed notification payload properties
```

Comment:
Is this doc somewhere we would typically leave examples too? I can see the request body being a little confusing to form, and an explicit example would be very helpful to developers.